webapp/src/types/model.ts

```diff
@@ -9,6 +9,27 @@ export type AllowedValue = {
     name?: string;
 };
 
+export enum JiraFieldCustomTypeEnums {
+    SPRINT = 'com.pyxis.greenhopper.jira:gh-sprint',
+    EPIC_LINK = 'com.pyxis.greenhopper.jira:gh-epic-link',
```

Comment:
Was the moving around of these types in the file caused form the `no-use-before-define` rule?

Reply:
I think we should let types be used before they were defined as it was in this file before, which is why I'm suggesting to disable `no-use-before-define` in favor of `@typescript-eslint`'s version of the rule, which allows types to be defined out of order. I presume the order of the types in that file were chosen for readability (though may be more random than that)

Either way, I think we should preserve the source files and have the linter allow for this case